src/libraries/System.Runtime/README.md

```diff
@@ -0,0 +1,16 @@
+# System.Runtime
+Contains fundamental classes and base classes that define commonly-used value and reference data types, events and event handlers, interfaces, attributes, and processing exceptions.
+
+Documentation can be found here: https://learn.microsoft.com/en-us/dotnet/api/system?view=net-7.0.
+
+## Contribution Bar
+- [x] [We consider new features, new APIs and performance changes](../../libraries/README.md#primary-bar)
+
+See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-System.Runtime+label%3A%22help+wanted%22+) issues.
+
+## Source
+* [../../coreclr/System.Private.CoreLib/src/System](../../coreclr/System.Private.CoreLib/src/System)
```

Comment:
Majority of the implementation is under libraries.

The runtime-specific paths have all coreclr, mono and nativeaot variants.